.github/actions/kots-e2e/action.yml

```diff
@@ -0,0 +1,110 @@
+name: 'KOTS E2E'
+description: 'Composite action for running KOTS e2e test'
+inputs:
+  test-focus:
+    description: 'Name of the test for FOCUS'
+    required: true
+  kots-namespace:
+    description: 'KOTS namespace'
+    required: true
+  k8s-version:
+    description: 'K8s version'
+    required: true
+  testim-access-token:
+    description: 'Testim access token'
+    required: true
+  kotsadm-image-registry:
+    description: 'Kotsadm image registry'
+    default: 'ttl.sh'
+  kotsadm-image-namespace:
+    description: 'Kotsadm image namespace'
+  kotsadm-image-tag:
+    description: 'Kotsadm image tag'
+    default: '2h'
+  testim-branch:
+    description: 'Testim branch'
+    default: 'master'
+  aws-access-key-id:
+    description: 'AWS access key id for uploading support bundle'
+  aws-secret-access-key:
+    description: 'AWS secret access key for uploading support bundle'
+  aws-region:
+    description: 'AWS region for uploading support bundle'
+    default: 'us-east-1'
+  go-version:
+    description: 'Go version'
+  node-version:
+    description: 'Go version'
+# outputs:
+#   random-number:
+#     description: "Random number"
+#     value: ${{ steps.random-number-generator.outputs.random-number }}
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/setup-go@v2
+      with:
+        go-version: ${{ inputs.go-version }}
+
+    - uses: actions/setup-node@v2
+      with:
+        node-version: ${{ inputs.node-version }}
+
+    - uses: replicatedhq/action-k3s@main
+      id: k3s
+      with:
+        version: ${{ inputs.k8s-version }}
+
+    - name: install ginkgo
+      run: go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo
+      shell: bash
```

Comment:
I know we specify shell in other places, but since bash is the default, would it be possible to standardize on only adding shell where we don't use bash?

Reply:
in an action you are required to specify the shell